guibot/controller.py

```diff
@@ -748,18 +749,16 @@ def mouse_click(
 
         See base method for details.
         """
-        toggle_timeout = GlobalConfig.toggle_delay
-        click_timeout = GlobalConfig.click_delay
         button = self._mousemap.LEFT_BUTTON if button is None else button
         if modifiers is not None:
             self.keys_toggle(modifiers, True)
         for _ in range(count):
             # BUG: the xdotool click is too fast and non-configurable with timeout
             # self._backend_obj.run("click", str(button))
             self.mouse_down(button)
-            time.sleep(toggle_timeout)
+            time.sleep(self.params["control"]["toggle_delay"])
             self.mouse_up(button)
-            time.sleep(click_timeout)
+            time.sleep(self.params["control"]["click_delay"])
         if modifiers is not None:
             self.keys_toggle(modifiers, False)
```

Comment:
Don't we have one more case of type_delay in this class that was missed?

Reply:
on this line?

Reply:
No, *in this class*

Reply:
Ok, we don't have control over the XDoTool regarding setting delay_between_keys.